lib/4_filter_numbers.rb

```diff
@@ -5,5 +5,12 @@
 # Run `bin/check` from the command line to execute the automated tests.
 
 def filter_numbers(arr)
-  # Write your code here.
+  arr.each { |x|
```

Comment:
Some syntax might help here. I don't think this is a hard-and-fast rule, but I usually see `{}` used for blocks on a single line and `do end` used for multiline blocks. Here, especially since you are nesting a conditional inside the block, I'd argue for using `do end`.

```ruby
arr.each do |x|
  if x>= 10
    arr.delete(x)
  end
end
```

That being said, there is at least one one-line approach that doesn't use `#each`, if you are interested.